common/lib/xmodule/xmodule/library_content_module.py

```diff
@@ -191,13 +191,15 @@ def _publish_event(self, event_name, result, **kwargs):
             "result": result,
             "previous_count": getattr(self, "_last_event_result_count", len(self.selected)),
             "max_count": self.max_count,
+            "user_id": user_id,
+            "course_id": unicode(self.course_id),
```

Comment:
I definitely do. When I did my first pass at this I forgot about needing the course id in the context even if it is also in the payload.